model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -435,8 +436,16 @@ def _create_model_run_config(
         model_config_variant: ModelConfigVariant,
         trial_objectives: TrialObjectives,
     ) -> ModelRunConfig:
+        trial_batch_size = (
+            int(trial_objectives["batch_sizes"])
```

Comment:
batch_sizes implies a list or multiple values. Is this always a scalar?
Why does it need to be cast to an int? Is the value coming out of trial_objectives a float?

Reply:
The cast is to pass static type checking (it could be a `str | int`, but in this case we know it's an int).
`batch_sizes` is what the parameter is called in the config (even though it's a scalar value)